examples/forecasts/basic_example_forecasting.py

```diff
@@ -0,0 +1,56 @@
+from forecasting.models.gluonts_model.mqcnn.mqcnn_model import MQCNNModel
+from forecasting.task.forecasting.dataset import TimeSeriesDataset
+
+import core as ag
+from core.scheduler import FIFOScheduler
+
+dataset = TimeSeriesDataset(
+    train_path="./COV19/processed_train.csv",
+    test_path="./COV19/processed_test.csv",
+    index_column="name",
+    target_column="ConfirmedCases",
+    time_column="Date")
+
```

Comment:
Can we define variables: 
`train_data = dataset.train_data`
`validation_data = dataset.test_data`

and use these instead `train_fn()` instead?  

As I've said multiple times, it's improper to tune hyperparameters using test-data, so we need to call it something else.

Reply:
I've changed almost every test_data in the implementation to val_data, except for those in the Timeseriesdataset, do we want to have train data, test data and val data in it at the same time?